modules/ROOT/pages/bonita-bpm-overview.adoc

```diff
@@ -1,12 +1,23 @@
-= Bonita platform overview
+= Bonita components
 :description: Bonita is an open-source and extensible platform for business process automation and optimization. The Bonita platform accelerates development and production with clear separation between capabilities for visual programming and for coding.  BPMN graphical notation, full extensibility, and reusable components allow smooth collaboration among the different profiles on the IT team, and with the business teams. Bonita integrates with existing information systems, orchestrates heterogeneous systems, and provides deep visibility into processes across the organization.
 
 Bonita is an open-source and extensible platform for business process automation and optimization. The Bonita platform accelerates development and production with clear separation between capabilities for visual programming and for coding.  BPMN graphical notation, full extensibility, and reusable components allow smooth collaboration among the different profiles on the IT team, and with the business teams. Bonita integrates with existing information systems, orchestrates heterogeneous systems, and provides deep visibility into processes across the organization.
 
 The Bonita Platform has two components: the development environment, Bonita Studio, and the runtime environment, Bonita Runtime.
 
+[.card-section]
 == Bonita Studio
 
+[.card.card-index]
+--
+xref:bonita-studio.adoc[[.card-title]#Bonita Studio# [.card-body.card-content-overflow]#pass:q[After reading what follows, you may want to know more]#]
```

Comment:
I think we should never write something like "Read what comes after", it feels weird and doesn't provide any information to the user.

```suggestion
xref:bonita-studio.adoc[[.card-title]#Bonita Studio# [.card-body.card-content-overflow]#pass:q[Develop business process automation applications using Bonita Studio]#]
```

Reply:
changed into something different